source/api/discovery/0.5/index.md

```diff
@@ -662,39 +676,39 @@ The new location of the IIIF resource, after it was affected by a `Move` activit
 
-Activities _SHOULD_ have the `endTime` property.  The value _MUST_ be a datetime expressed in UTC in the [xsd:dateTime][org-w3c-xsd-datetime] format.
+Activities _SHOULD_ have the `endTime` property. The value _MUST_ be a datetime expressed in UTC in the [xsd:dateTime][org-w3c-xsd-datetime] format.
 
-```
+```json-doc
 { "endTime": "2017-09-21T00:00:00Z" }
 ```
 
 ##### startTime
 
 The time at which the Activity was started.
 
-Activities _MAY_ have the `startTime` property.  The value _MUST_ be a datetime expressed in UTC in the [xsd:dateTime][org-w3c-xsd-datetime] format.
+Activities _MAY_ have the `startTime` property. The value _MUST_ be a datetime expressed in UTC in the [xsd:dateTime][org-w3c-xsd-datetime] format.
 
-```
+```json-doc
 { "startTime": "2017-09-20T23:58:00Z" }
 ```
 
 ##### summary
 
-A short textual description of the Activity. This is intended primarily to be used for debugging purposes or explanatory messages, and thus does not have the requirement for internationalization of the content. The intended audience of this field is a software engineer looking at the data, not an end user of the IIIF resource. It is helpful, but not even recommended that the field be present as in most situations, it will never be seen.
+A short textual description of the Activity. This is intended primarily to be used for debugging purposes or explanatory messages, and thus does not have the requirement for internationalization of the content. The intended audience of this field is a human who manages an aggregation service and is looking at the data, not an end user of the IIIF resource. It is not recommended that this field be present as in most situations it will never be seen.
 
-Activities _MAY_ have the `summary` property.  The value _MUST_ be a string.
+Activities _MAY_ have the `summary` property. The value _MUST_ be a string.
 
-```
+```json-doc
 { "summary": "admin updated the manifest, fixing reported bug #15." }
 ```
 
 ##### actor
 
 The organization, person, or software agent that carried out the Activity.
 
-Activities _MAY_ have the `actor` property.  The value _MUST_ be a JSON object, with the `id` and `type` properties. The `id` _SHOULD_ be an HTTP(S) URI. The `type` _MUST_ be one of `Application`, `Organization`, or `Person`.
+Activities _MAY_ have the `actor` property. The value _MUST_ be a JSON object, with the `id` and `type` properties. The `id` _SHOULD_ be an HTTP(S) URI. The `type` _MUST_ be one of `Application`, `Organization`, or `Person`.
```

Comment:
Just confirming: no `name` property?  This is inconsistent with https://www.w3.org/TR/activitystreams-vocabulary/#actor-types

Reply:
Pretty sure we had `name` and took it out. Can't add back in before 0.9.1 if it is needed, regardless. Should open an issue to track.

Reply:
I don't have an opinion about having it or not, was just noting the divergence from the AS spec examples